lib/android.js

```diff
@@ -621,7 +629,9 @@ function _getArtClassLinkerSpec (api) {
   for (let offset = startOffset; offset !== endOffset; offset += pointerSize) {
     const value = classLinker.add(offset).readPointer();
     if (value.equals(internTable)) {
-      const delta = (getAndroidApiLevel() >= 23) ? 3 : 5;
+      var delta = 5;
+      if(getAndroidApiLevel() >= 23) delta = 3;
```

Comment:
```suggestion
      if (getAndroidApiLevel() >= 23) {
        delta = 3;
      }
```

This module follows [semistandard](https://github.com/standard/semistandard).

But let's move these computations outside the loop, and only call `getAndroidApiLevel()` once.

Reply:
(Actually let's not move them, it will make it harder to see what changed. Should be refactored in a separate PR.)

Reply:
We should only call `getAndroidApiLevel()` once though, since we need to change that line anyway. E.g.: `const apiLevel = getAndroidApiLevel();`